src/sso/sso_auth_session.h

```diff
@@ -66,6 +44,8 @@ class SSOAuthSession {
   virtual ~SSOAuthSession();
 
   int jsid() const { return jsid_; }
+  SignonAuthSession* session() const { return session_; }
+  std::string method() const { return method_; }
```

Comment:
Look at where you use this getter, I'd suggest to define as `const std::string& `